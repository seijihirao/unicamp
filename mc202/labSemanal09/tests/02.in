((()2:6.580(()3()))9.490:1:7.620(()4()))
(()1:2.070((()3:9.820(()4()))2.190:2()))
(()1:4.720(()2:6.870(()3:9.220(()4()))))
((((()4())6.630:3())3.190:2())0.760:1())
