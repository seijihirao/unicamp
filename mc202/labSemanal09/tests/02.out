16.070
14.080
20.810
10.580
